pipelines/rj_smtr/tasks.py

```diff
@@ -576,7 +593,7 @@ def upload_logs_to_bq(
     create_or_append_table(
         dataset_id=dataset_id, table_id=table_id, path=filepath.parent.parent
     )
-    if error is not None:
+    if error is not None and recapture is False:
```

Comment:
precisa desse `and recaptura is False` ?

Reply:
```suggestion
    if error is not None:
```
sempre que houver um erro deve ocorrer um `raise` aqui